src/ORM/Table.php

```diff
@@ -2304,4 +2305,26 @@ public function __debugInfo()
             'connectionName' => $conn ? $conn->configName() : null
         ];
     }
+
+    /**
+     * Validates that a belongsToMany or hasMany assoc has at least one entry.
+     *
+     * @param mixed $value
+     * @return boolean
+     */
+    public function validateAtLeast($value, $expectedCount = 0, $operator = '>')
```

Comment:
Honestly, I don't care, just tell me what I need to do to get it merged and I'll do it if it's not totally against what I had in mind.